### PR TITLE
Virologist job is gone/Heal Virus is gone/Viro lab is still there

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I can't think of a way that passive, unlimited, free, no downside health regen will ever be good gameplay.

I can't think of a way in which healing viruses will ever be anything but that.

And I can't think of a reason to stick non antagonists alone in an r wall box for 40 minutes if they can't do anything beneficial in an average round.

The virology lab/virus system in general is interesting so the lab is still there.

I'd be happy to be talked down from this/hear alternate proposals because otherwise I know lots of people will be mad at me but we've had virology for something like 6-7 years now and nobody has ever figured out how to make the job interesting for loyal crew without absurd beneficial viruses.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
inb4 people telling me how being able to ignore environmental hazards and almost never need to interact with another player for healing again after 5 minutes into the round is actually excellent design and passive no brainer bonuses are actually very interesting for our game about surviving disaster

**Comment by Repukan** [comment]
First

**Comment by ChangelingRain** [comment]
actually, heal viruses are good for the game since they let you walk through lava without dying.

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -1,329 +0,0 @@
...
-
-Bonus
-	Heals burn damage over time, and helps stabilize body temperature.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/heal/burn/plus
-
-	name = "Temperature Adaptation"
```
Comment (praisenarsie) [inline code/datums/diseases/advance/symptoms/heal.dm]: at least keep this

Comment (ChangelingRain) [reply code/datums/diseases/advance/symptoms/heal.dm]: actually, being able to walk through space is good for the game, since we don't already have a job that lets you do that.

Comment (Jalleo) [reply code/datums/diseases/advance/symptoms/heal.dm]: We have a alternativefor this its in genetics.

Comment (ChangelingRain) [reply code/datums/diseases/advance/symptoms/heal.dm]: actually, being able to walk through space is good for the game, since we don't already have a job that lets you do that.

**Comment by Celau** [comment]
what about genetics?? why virology??

File: code/__DEFINES/jobs.dm [hunk 2]
```diff
@@ -22,7 +22,6 @@
 #define CMO_JF				(1<<3)
 #define DOCTOR			(1<<4)
 #define GENETICIST		(1<<5)
-#define VIROLOGIST		(1<<6)
```
Comment (Jalleo) [inline code/__DEFINES/jobs.dm]: Since you are removing the virologist can you add one doctor role to balance out how many there are for the department? (At times the virologist basically acts as a normal doctor)

**Comment by Spaceinaba** [comment]
@Celau because you can actually kill a geneticist

**Comment by Jalleo** [comment]
@Celau Kor has chosen to remove one. Maybe we will get both removed but that isnt to do with this PR.

**Comment by Spaceinaba** [comment]
I doubt genetics will get removed
now hulk on the other hand

**Comment by ChangelingRain** [comment]
nothing in genetics is good for the game anyway, infinite spacewalking, ability to ferret out antags through walls, ability to fight people from across the screen, and stun immunity are all terrible for the kind of game we have here

**Comment by Buggy123** [comment]
:-1: I for one, have fond memories of turning out triple stacked healing stim viruses within 5 minutes. Good times, good times...

**Comment by Bawhoppen** [comment]
Definitely a needed change :+1: 

Though I'd wish we could keep the job

**Comment by uraniummeltdown** [comment]
Why don't we port Bay / /vg/ Virology?

**Comment by Qustinnus** [comment]
I hate virology but flat out removing it sounds stupid.

**Comment by swindly** [comment]
improve don't remove

**Comment by Qustinnus** [comment]
also for a proposal; make more innocent viruses that can only be resolved by virologist made medicine (so not fucking common household ingredients like sugar) that can be received from sources of filth like blood, vomit, rats, cuts etc. 

maybe if not every feature on TG was focused on creating new ways of powergaming we wouldn't have issues with jobs only being used to powergame.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
You need to remove/change viro access requirements on the maps

**Comment by KorPhaeron** [comment]
@Cyberboss The virology lab is still there though, and its a distinct room the CMO can still access and that ID computers can still grant/remove access for

**Comment by Qustinnus** [comment]
why not just make it CMO access if the CMO is the only one to use it and you don't want virology used as a job

**Comment by Cyberboss** [comment]
Changelog

**Comment by Jalleo** [comment]
Its legacy support to make it easier to add back in as a better way.

**Comment by Cyberboss** [comment]
Wouldn't giving doctors viro access by default allow for more antag opportunities?

**Comment by Repukan** [comment]
Where is the PR to remove genetics too.

It's free powers

**Comment by XDTM** [comment]
👍 if you make all doctors able to access virology and work in it
👎 for removing healing without bothering to think of an alternative

**Comment by Repukan** [comment]
You can steal heal though?

**Comment by imsxz** [comment]
He did bother to think for an alternative, if you bothered to read the full PR. How about you bother to suggest an alternative, as he stated he was open to suggestions in the PR.

Perhaps this is outside of the scope for the PR, but maybe adding a system that makes diseases with good symptoms prone to mutating into other diseases? Like a super healing virus could give you prions or something, just an example.

**Comment by XDTM** [comment]
A possible idea is to make symptoms spawn as blocks of 2-3, who are an all-or-nothing deal; you can roll the healbrute-sneezing combo, if you're lucky, or maybe a Dna Restoration | Flesh Death | Burn Healing, and consider if it's worth adding. Virus length would probably be measured in blocks rather than symptoms, so viros would have to choose between more chance of bad symptoms or less symptoms overall.

**Comment by PKPenguin321** [comment]
This is a bit ham fisted and should be postponed until we at least give people a chance to think of alternatives to just "free healing" that would make the job worth keeping

**Comment by KorPhaeron** [comment]
@PKPenguin321 7 years!!

**Comment by PKPenguin321** [comment]
We've survived with it for 7 years we can take another couple days to let the idea guys take a shot at it

**Comment by TribeOfBeavers** [comment]
I don't think removing the healing is necesarily a bad thing, but the entire job I'm not sure about.

Adding a bunch of minor positive/neutral/cosmetic symptoms is probably worth looking at, right now it's basically an antag only thing without healing which is a bit of a shame.

Also, giving viro access to MDs could be a good idea. If viro is CMO only the only time it'll be used is when the CMO is a traitor. I doubt most players would give out viro access to others who ask if it can only make deadly or annoying viruses.

**Comment by GuyonBroadway** [comment]
I use Viro to infect the crew with a virus that makes them bald and black. It really pisses off the snowflakes.

**Comment by PKPenguin321** [comment]
obesity viruses would be top tier for that same if we had fatsprites

**Comment by Nabski89** [comment]
Idea XDTM
You can never get healing without some other negative symptoms. MAYBE these can be mitigated with RNG and interaction with other crew members so it's still unlimited free healing which is bad.

Idea 1
Maybe make the virus's have activation requirements. (Shadowlings in dark, Podpeople light, Vampire coffins)

Brute is only healed if you've got sugar in your system
Burn if you're tripping on drugs/drunk
Toxin if you're hungry
Oxygen if you're cold

Idea 2
Damage isn't healed, it's just mitigated

Brute damage becomes burn
Burn becomes Toxin
Toxin becomes oxygen
This is slightly still healing if you're breathing, but puts a much harder limit on how fast you heal.

**Comment by XDTM** [comment]
I really like both ideas!

**Comment by GuyonBroadway** [comment]
Make it consume nurtiment to healing and imply that its because your bodies cells go into overdrive and need the fuel for rapid self repair.

**Comment by XDTM** [comment]
then again hunger at worst slows you a bit, so it's not the best balancing mechanic until we make it have consequences

**Comment by GuyonBroadway** [comment]
No not nutrition the actual nutriment chem. So you need to keep eating to fuel the regeneration and can't cheese it out by adding weight gain to your virus.

**Comment by Alecw155** [comment]
I have 0 qualms about losing healing virus. What are the biggest problems people have with Virology as a job? 

- The fact that they're separated from the crew for a majority of the round? 
I would say either position Virology closer to Medbay/less sealed off (although I suppose that might defeat the point of trying to keep diseases from spreading) or forcing the virologist to come out of their hole more often to, perhaps, get more materials or something. 

- Lack of usefulness outside of metagaming?
Maybe add some more fringe-case diseases/viruses, probably non-lethal but still debilitating, like stuff you can only pick up from specific locations/activities (lavaland diseases, a virus you can only contract by being attacked by space carp, a fungal disease contracted from botany, etc) to make people more dependent on Viro. Would that be too much of the pain in the ass to have to worry about in a given round?

**Comment by Qustinnus** [comment]
add moodies then we can use them for viruses

**Comment by nicbn** [comment]
Just remove symptoms like tissue regrowth so that the only way to make a regen virus is by making both toxic compensation and toxic filter. I mean, viro used to be like that and it worked.

**Comment by ExcessiveUseOfCobblestone** [comment]
What if you just removed healing viruses and have low-tier diseases be a fairly common event so the job is still actually viable?

Later we can add sterility mechanics and **move viro from reinventing the powergamewheel (self-healing) to mitigating/treating something that can't be immediately fixed with common medical items**. I'm thinking as the round progresses diseases can appear more frequently and more deadly if you're in nonsterile/dirty areas (janitor's end), but can be mitigated on viro's end by isolating symptoms and distributing antibodies across the station (lessens effects of that symptom) along with their current job of getting antibodies for the specific virus while not making it apparently obvious they're making a deadly disease (which they can also distribute since you will be expecting them to give you antibodies).

**Comment by praisenarsie** [comment]
regeneration is the most op heal sympton because a virologist will just go up to you and ask what's brute damage.

**Comment by SyruseEln** [comment]
if youre going as far as to remove the job just remove all aspects of  it entirely.

virology is OP for both helpful and harmful viruses and leads to really awful experiences for both.

a better option would be to rework the advanced disease system entirely to make getting game-ending viruses both harder to get and less effective to obtain, but good luck doing that because creativity is hard.

**Comment by SyruseEln** [comment]
oh wait youre removing healing viruses but not death viruses. yeah no way fuck this change. just rip the whole fucking thing out instead of introducing another greif-only role

**Comment by Durkel** [comment]
Good riddance. For years it was a completely useless job until gia reworked it inot a powergame factory. From there it's only got worse over the years.

One job shouldn't be able to make an entire department irrelevant.

**Comment by SgtxRootbeer** [comment]
Remove weight balance, and make the healing symptoms starve you, and outright stop if you are too hungry.

Yes, this can be cheesed via nutriment pumps. That requires Dept interaction between viro, rnd, robotics, and mining so leave it.

**Comment by ExcessiveUseOfCobblestone** [comment]
If the issue is self-healing at all, why would self-healing but downsides that you can also entirely negate make that ok?

**Comment by Durkel** [comment]
Reminder that HG was right and did nothing wrong.

#14239

**Comment by Nabski89** [comment]
Cobby, the issue isn't self healing at all, but self healing with no required further thought or effort required. On another pointless note I like that virologist as a job being merged with medical doctor.

**Comment by ike709** [comment]
Virologist is the only medical job I'm willing to play on TG.

**Comment by Alecw155** [comment]
Can't we just take out the extreme ends of the spectrum of healing/deadly symptoms and make diseases on the whole more "very inconvenient" than life-threatening? I'd rather try to improve a feature that has been around for years than rip it out entirely because there's no immediate, obvious alternative

**Comment by Repukan** [comment]
You've had 7 years to improve it.

**Comment by PKPenguin321** [comment]
Not an argument

**Comment by Alecw155** [comment]
I didn't know we've hit the non-existent deadline for changing old features in a spessman game

**Comment by ninjanomnom** [comment]
After a period of time where it's clear no one wants to improve or maintain a feature and it's fallen behind in terms of quality, yeah removal is the best option. 

This isn't even a full removal.

**Comment by WJohn** [comment]
i find the entire concept of diseases to be mechanically uninteresting and unfun to play with and to play against. it's like invisible poison that spreads and does awful shit to you like removal of control (stunlock vomit, retardation etc). this is fundamentally what diseases even *are*, and if you want them not to spread, then you've just got a plain old poison of which we have countless varieties of already.

i'd say throw the whole entire concept in the trash. i don't think anything can even be salvaged from it, as there is absolutely nothing fun or engaging about it worth keeping.

**Comment by XDTM** [comment]
> After a period of time where it's clear no one wants to improve or maintain a feature and it's fallen behind in terms of quality, yeah removal is the best option.
> 
> This isn't even a full removal.

Dude i reworked it like a month ago

Yes it ended up unbalanced which is why i kept asking for feedback to adjust it, not silence and then a sudden kor removal PR

**Comment by Arzion5** [comment]
>UNIRONICALLY TRYING TO REMOVE THE DEDICATED VIROLOGIST JOB BECAUSE OF GOOD VIROLOGISTS THAT HEAL PEOPLE
>UNIRONICALLY TRYING TO REMOVE HEALING SYMPTOMS BECAUSE OF GOOD VIROLOGISTS THAT HEAL PEOPLE
>IMPLYING THIS WON'T LEAD TO A TOTAL VIROLOGY REMOVAL IN THE FUTURE BECAUSE LITERALLY NO ONE ELSE WILL USE VIROLOGY BECAUSE THE ONLY JOB DEDICATED TO IT WAS REMOVED
Was there even a poll?
>THIS MIGHT END UP MERGED DESPITE HAVING A 1:1 LIKE/DISLIKE RATIO ON THE PR

**Comment by KorPhaeron** [comment]
@arizon5

You made no arguments for why passive, no drawback unlimited free healing and space immunity is good for the game

You didn't really make any arguments at all other than a half hearted attempt at populism despite the "votes" being evenly split

**Comment by Arzion5** [comment]
I will explain.

1. The PR will greatly reduce the total amount of time Virology is used.
Virology is a very dull job as it is. You repeatedly do the same thing in trying to get lucky with the symptoms and/or removal of the already-present symptoms. For that reason, it is one of the least-played jobs on the server. Completely removing a job dedicated to this mechanic will just mean Virology will be even less used, and in the future this can lead to the removal of the Virology sub-department as a whole, just because it is almost never used and serves only for antagonists to fuck the station over with. With a job dedicated to Virology, it still gets a fair share of playtime from whoever is willing to play it, without being discouraged by the various access restrictions and the amount it takes to have your ID upgraded with Virology access. All the rounds where healing viruses were spread across the station were actually few in number BECAUSE virology is not played too often. If the PR passes, Virology will not only not have a job dedicated to it, but it will also be very unlikely that it will ever be used by anyone else, as it is still behind access restrictions, and the only time medical doctors MIGHT gain access to it is during skeleton crew rounds. Medical doctors will not take the mantle, nor the CMO, especially when the CMO is a busy head of staff with access to shiny toys that would better serve treating patients and the medical doctors are also busy themselves trying to heal injured people.

2. It will reduce the positive aspects of Virology.
Virology is an interesting job that can do equal harm and good. Removal of the healing viruses will damage the "good" aspect of Virology, making it more of a job where you just wish you rolled antag for to make use of all the nasty symptoms that you can acquire, because the good symptoms are much fewer in number. The PR also removes DNA Restoration and temperature stability, while flesh mending (an upgraded variant of the brute-healing symptom) turned clone damage into burn damage. The PR removes the entire healing symptom category, which are 7 (8 if you count the default "heal" symptom, 4 symptoms if the upgraded symptoms and the basic heal symptoms are not counted, I personally do not know if you can acquire the upgraded healing symptoms anymore) symptoms. This means that there will be roughly 20 negative symptoms available, 8 neutral symptoms and only **6** positive symptoms (**4** if you do not count Viral Evolutionary Acceleration and Viral Self-Adaptation, which are give-and-take symptoms).

3. The solution would be much simpler.
If people healing massive amounts of damage is a problem, then reduce the amount of healing that symptoms can produce, or add a limitation/cap on how much someone can heal from symptoms. Outright removing the virologist job and damaging the positive aspects of the diseases mechanic is what I assume to be an inefficient way of solving the problem.

4. It is already moderately difficult to acquire the healing symptoms.
Out of a pool of roughly 32 symptoms, with the current amount of approximatively 8-10 positive symptoms, it is a 1/3 to 1/4 chance to acquire a positive symptom. The usual 3 healing symptoms, out of 10 positive symptoms, means that there is a further 1/3 chance that ONE of the healing symptoms will be selected. That's already 8-12% of a healing symptom to appear each. Add further that there is a 1/3 chance of having duplicates of the healing symptom that you have acquired, which jumps to 2/3 chance of having a duplicate after you have acquired 2 of the healing symptoms. In short, it takes a lot of **luck** to roll the healing symptoms, and even more so to get all 3 of them within a respectable time-frame. **_The "space immunity" is merely the success of a virologist that has managed to create a powerful, great virus_**

5. Virology has already been balanced.
The removal of multiple diseases being present in your body and the subsequent virology rework has removed virology's status as a game-breaker, rewarding skill, luck, and proper symptom management with significant gameplay advantages. This PR merely adds "salt to the wound" and only serves to further damage virology as a valuable mechanic to "invest" in.

Tl;dr : Fun should be allowed.

**Comment by ExcessiveUseOfCobblestone** [comment]
I feel like if we make the symptoms come with downsides they'll be easily negatable so nothing really changes. It should have requirements that make it niche, not something that everyone can/ wants to just pick up.

**Comment by ExcessiveUseOfCobblestone** [comment]
Also above poster have you played the game (Viro) recently? Honest question

**Comment by Incomptinence** [comment]
All regeneration effects should have a chance to create tumours inside you when you heal that need to be surgically removed. 

Get too many you just die on the spot say one clump of deadpool'd regenerating cells gone tumour blocked an artery or something.

**Comment by KorPhaeron** [comment]
>The PR will greatly reduce the total amount of time Virology is used.

I agree, that is why it will become a CMO or med doctor thing


>It will reduce the positive aspects of Virology.

It's "positive" and desirable effect from a players perspective to have unlimited health sure, but again, I don't think such healing is positive or desirable for the game. If I could find something that gave me noclip in a platforming game or there was a magic infinite ammo box in PUBG  it'd be "positive" but it'd still degrade the experience overall

>The solution would be much simpler.

Make a PR then, but keep that it isn't just about how fast they heal, but that they are just passively better than the baseline for the rest of the game for no cost or upkeep.

>It is already moderately difficult to acquire the healing symptoms.

People are regularly getting this before the 15 minute mark. 5 minutes isn't uncommon either.

>Virology has already been balanced.

Gonna have to echo Cobbys question of whether you've played on /tg/ recently if you think a 12/12 healing virus at 10 minutes into the round is balanced.

**Comment by Arzion5** [comment]
>I agree, that is why it will become a CMO or med doctor thing

That is the problem, CMOs and Medical Doctors won't pay attention to it. To quote : 
"Medical doctors will not take the mantle, nor the CMO, especially when the CMO is a busy head of staff with access to shiny toys that would better serve treating patients and the medical doctors are also busy themselves trying to heal injured people."

>It's "positive" and desirable effect from a players perspective to have unlimited health sure, but again, I don't think such healing is positive or desirable for the game. If I could find something that gave me noclip in a platforming game or there was a magic infinite ammo box in PUBG it'd be "positive" but it'd still degrade the experience overall

Your mistake is that you compare innate healing with extremes in other videogames. It is not "unlimited health", it is simply just something that heals your damage over time, perhaps one of the **most common videogame character abilities.** If innate healing has become such a massive issue, then the simple solution is to reduce said healing or add a limit to it, which leads to...

>Make a PR then, but keep that it isn't just about how fast they heal, but that they are just passively better than the baseline for the rest of the game for no cost or upkeep.

"Make a PR" is a very shifty excuse.
"I assume the way you're doing it is wrong, I recommend doing X instead because it is advantageous."
"Do X yourself, I am not changing my Y."
Not every positive thing needs to have a drawback or a monkey paw twist. Again, the simplest solution is to either reduce the health regeneration overall or add a cap. For all the progressive jobs, you have : 
- Science. Throw items into a destructive analyzer to unlock new items, fabricate very useful items and upgrades of already-existing items. There is a PR going on that makes the advanced energy gun emit radiation, which would make it **the first R&D item to have a drawback.**
- Genetics. Constantly irradiate someone's genes to unlock new abilities and disabilities. The only thing with a drawback is hulk, which is not being able to use items, but the advantages are very powerful compared to the disadvantage. Didn't stop people from whining about hulk and trying to go down the same path as multiple other servers that have tried to make hulk as useless as possible without ruining its essence.
- Mining, weirdly enough. There is a lot of loot to be gained that has no drawback, but it is balanced by the fact that it is very dangerous to acquire said loot. The closest thing to a drawback is a miner turning into an ash lizard, because it means they cannot wear footwear, but they do not need to breathe air.

>People are regularly getting this before the 15 minute mark. 5 minutes isn't uncommon either.

That's just skill and luck, for the same reason people can find all 6 power genes while doing Genetics in less than 20 minutes. There is nothing preventing a person from repeatedly spamming virus food, mutagen or liquid plasma on a disease beaker.

>Gonna have to echo Cobbys question of whether you've played on /tg/ recently if you think a 12/12 healing virus at 10 minutes into the round is balanced.

The point is that outright ruining an entire sub-department and killing off half of the beneficial effects that diseases can give **JUST BECAUSE** people have been running around with innate regeneration is downright unwise. As I have recommended, either reduce the amount of healed damage from symptoms or add a cap.

**Comment by ExcessiveUseOfCobblestone** [comment]
I asked because he implied it's balanced around Rng of 20something symptoms when there is a whole section of the wiki dedicated to showing individuals how to largely remove the rng

**Comment by KorPhaeron** [comment]
Most other straight upgrades in the game either have risk inherent in obtaining them or a resource cost. (Genetics is also not good)

It isnt a monkey paw twist to have to decide if you want to spend your diamonds on a mech or a gun or a bag of holding. It isnt a monkey paw to decide if X loot is worth the inventory slot vs wearing Y instead

>advanced egun will be the first with a drawback

It was the first back in like 2010 because it was the only gun in RD when it was added and had that drawback but that is no longer the case. Tesla guns can shock you, beam rifles require windup time, mechs are slow/use battery/have no hands, etc

>doctors/cmo will not do it often

Okay? Thats why it isnt its own job, it wont have enough to warrant being done every round anyway.

>regenerating health is common in videogames

Regen health in CoD relates to SS13 how?

>its shifty for me to set deadlines and goals as the person responsible for doing so

??

Nobody has been able to strike a balance between viro being not worth playing and viro breaking the game in seven years. Either this PR inspires someone to fix it or the virus is gone, either way the game will be improved

>outright ruining half a department

If taking away the heal virus ruins viro it isnt worth saving in its current state

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm a bit hesitant you've beaten around the bush for a simple yes/no question of whether or not you've played the game. Considering at least some of your arguments are half-truths such as the one mentioned above, I'm beginning to assume you aren't fully aware of current viro mechanics or at worse purposefully misrepresenting.

**Comment by MrStonedOne** [comment]
Honestly, the best way to do this is to remove the viro lab, remove the need for a viro lab, and make it so you can make viruses with a variety of jerry rigged tool combos. Like maybe you have to find some germ cultures in blood/dried blood/ or irradiated/spoiled food around the station, Use a source of radiation to enhance it, feed it milk or some other shit to grow it. use a health scanner you jerry rigged up with a nano manipulator, Basically make the micro organism version of botany, and remove the snowflake machines

This fits within ss13's theme, makes it possible to use for both good and bad without gating it behind some snowflake machinery, and lets us turn the existing viro labs into hermetically sealed isolation chambers for that sweet sweet lore.

**Comment by Arzion5** [comment]
@KorPhaeron 
>Most other straight upgrades in the game either have risk inherent in obtaining them or a resource cost. (Genetics is also not good)

Genetics is actually a good department. It fits within SS13's futuristic theme by having people be "upgraded" in various ways, while also providing powerful advantages.

>It isnt a monkey paw twist to have to decide if you want to spend your diamonds on a mech or a gun or a bag of holding. It isnt a monkey paw to decide if X loot is worth the inventory slot vs wearing Y instead

Exactly.

>Okay? Thats why it isnt its own job, it wont have enough to warrant being done every round anyway.

So you want to remove the virologist job to discourage people from using virology mechanics?

>Regen health in CoD relates to SS13 how?

It's not limited to just SS13, it's a common feature in many games, not just CoD. I have made the connection between health regeneration in videogames and health regeneration in SS13 like how you made the connection between cheats in videogames and health regeneration in SS13.

>its shifty for me to set deadlines and goals as the person responsible for doing so
??
Nobody has been able to strike a balance between viro being not worth playing and viro breaking the game in seven years. Either this PR inspires someone to fix it or the virus is gone, either way the game will be improved

Your plan is to ruin a sub-department and a big part of the positive virology mechanics JUST so that someone else will fix it? That's extremely counter-intuitive.
Putting words in my mouth is not good, I have never mentioned deadlines anywhere and the particular thing that I have mentioned to be a shifty excuse is to tell the other person to make the PR.

>If taking away the heal virus ruins viro it isnt worth saving in its current state

As I have previously mentioned, reduce the health regeneration or add a cap.

@MrStonedOne Your idea is best suited only when the virology sub-department also exists. Turning Virology into a makeshift-only mechanic with no dedicated department would not be feasible, it would waste a lot of the virology mechanic's potential.

@ExcessiveUseOfCobblestone I have not played as a virologist in a while, but I have documented myself well enough to at least know what I am talking about. If you have any concerns about my arguments, do not be afraid to express them, I will answer them.

**Comment by ChangelingRain** [comment]
>I have made the connection between health regeneration in videogames and health regeneration in SS13 like how you made the connection between cheats in videogames and health regeneration in SS13.

not only does this sentence not make any sense, it fails to consider how other games restrict health regeneration effects, if they have them at all.
most games don't just have infinite regeneration; it's usually limited in some form(only after getting hit, only a set amount, only in limited locations, etc)

**Comment by KorPhaeron** [comment]
>genetics is actually a good department

Sitting in front of a machine alone rolling dice until you can see through walls and are space immune and your hitbox is halved and then you hand out those advantages to everyone on station for free isn't very exciting gameplay either

>health regeneration in SS13 like how you made the connection between cheats in videogames and health regeneration in SS13.

Context matters. The concept of cheats breaking a videogame is a universal one. Health regen being a cheat vs a feature is entirely dependent on the setting. It works in Halo, it'd gut half of Rimworld.

>Your plan is to ruin a sub-department

You misunderstand, it's already ruined, and it's negatively impacting the rest of the game. People never needing to eat is not interesting for the chef. People never needing to heal is bad in combat and further undermines medbay/cargo/RnD/everyone else involved in maintaining medical supplies and machinery.

Things don't just exist in a vacuum. For an (outdated) example, we removed away missions for two years because healing viruses allowed people just to facetank all of them.

**Comment by ChangelingRain** [comment]
@Arzion5 what games have infinite, constant, rapid, unremovable, no-upkeep regen as anything other than an endgame/you've basically won now have fun state?

File: code/__DEFINES/jobs.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/__DEFINES/jobs.dm]: @KorPhaeron this pls

**Comment by Iamgoofball** [comment]
lmao free heal memelords go home

File: code/__DEFINES/jobs.dm (hunk 2 above)
Comment (KorPhaeron) [reply code/__DEFINES/jobs.dm]: Giving people a chance to make alternative PRs before I put more work into this one but I will

**Comment by Arzion5** [comment]
@ChangelingRain This gave me a better idea to recommend. Instead of outright ruining Virology, how about adding a time limit? Several ticks without taking any damage and the person will regenerate his health. If the person has taken damage, the person will either not heal or heal at a reduced rate.

@KorPhaeron 
>Sitting in front of a machine alone rolling dice until you can see through walls and are space immune and your hitbox is halved and then you hand out those advantages to everyone on station for free isn't very exciting gameplay either

Hey, as long as it works. The geneticist worked through the RNG and they got rewarded, and can imbue the rewards to everyone else.

>Context matters. The concept of cheats breaking a videogame is a universal one. Health regen being a cheat vs a feature is entirely dependent on the setting. It works in Halo, it'd gut half of Rimworld.

You are correct, healing is context-based, but in a game like Space Station 13, it shouldn't be that big of a deal, especially when you can already apply healing patches to yourself, which heal every part of the body and insert a chemical in you that further heals the damage over time.

>You misunderstand, it's already ruined, and it's negatively impacting the rest of the game. People never needing to eat is not interesting for the chef. People never needing to heal is bad in combat and further undermines medbay/cargo/RnD/everyone else involved in maintaining medical supplies and machinery.
Things don't just exist in a vacuum. For an (outdated) example, we removed away missions for two years because healing viruses allowed people just to facetank all of them.

The game is not negatively impacted because of Virology. You could say the same thing about people not needing to eat if there are a large amount of food vendors around the station, even if the vendor food nutriments are low. The problem with the current PR is that it will downplay the positive qualities of virology and make it more of a thing that will only be done by antagonists who want to kill people with nasty viruses. Who is actually going to work on Virology for only almost a handful of positive symptoms?

In a slightly related note, the medical department and the science department can theoretically create supermen using the aid of virology symptoms, genetics and liberally inserting implants into them. This will create crewmembers that : 
- Automatically heal
- Have reduced stun times
- Won't give up their grip on their items
- Cannot be stunned
- Have telekinesis
- Can vent-crawl when naked
- Can smash walls
- Use tools from their arms
- Have no need for air masks to enable internals
- Have increased resistance to impure air and temperatures
- Can see through walls
- Have no need to eat
- Etc.
Fun for the whole family

**Comment by Jalleo** [comment]
@Arzion5 Are you fillibustering?
 People have trued a lot to improve virology but sadly it hasnt made itself better gameplay only help at powergaming or causing sneeze spam. In 5 short sentences prove to me how it can be better. I mean better as a interesting mechanic that has someone going about the station interacting with things and people for either good or bad effect. Than say staying in a locked down room for 30 mins or more.

**Comment by Nabski89** [comment]
@Jalleo 
You slot symptoms into and out of viruses like they are chemicals. In order to unlock a new symptom for use you have to get something to stage five and cured. You can use the original monkeys for this, but those will quickly run out as you can only use each test subject once. The quality of symptoms is locked behind how many test subjects/virus's you have done.

So basically the DNA vault but a bigger struggle.

Lvl 1: Same
Lvl 2: Memes
Lvl 3: Stat alters
Lvl 4: Small healing/damage effects
Lvl 5: More memes
Lvl 6: Things that could actually be useful during a fight, rather than three minutes later.

**Comment by Nabski89** [comment]
There are ways to increase the crew interaction.
There are ways to make it less "free healing forever".

I still support the job being merged with standard doctor.

**Comment by Jalleo** [comment]
Yiu fell into the bit where you basically have the virologist staying in virology. We dont want that in the game. Its not a imrpovement more a timed system you have described by what I read. Anyhow going to sleep now.

**Comment by Arzion5** [comment]
@Jalleo "If you disagree with me, then you are doing X"
"If you don't want someone to ruin Y then tell us how it can be improved"
I assume that your question is "trapped", because there is a very likely chance any suggestion I'd make for Virology will be laughed off, and if I admit that I do not know how to improve it, I will lose credibility as I "dared to talk against removing something if I do not know how to improve it". The reality is that I do indeed do not know how to improve Virology, the communities have their own ideas that they can pool together to improve a mechanic or add new ones. Outright removing half of the positive symptoms and removing the only dedicated job to it is what I perceive to be a wasteful way to proceed, as the only people that will go to Virology will most likely be antagonists that will try to make harmful viruses. I have repeatedly stated the simpler solution, either reduce the amount of healing, add a cap, or (as of recently) make recently damaged people have reduced regeneration or none at all.
@optimumtact Jesus Christ the JPEG

**Comment by Durkel** [comment]
Alright I got it, a solution to genetics and virology. Remove both and turn them into sci-fi equivalents of nanobots. New department for sci/med, program the bots to do what you want, dependent on tech webs. However the potency of the bots requires frequent injections in order to keep them controlled, without frequent injections they go rogue and break your body down eventually dusting you. More potent the effects of the bots are the more frequent injections you need.

Boom there you go. Big effects, big downside. You can also make them increase hunger or some shit but you can balance around the fact that if you want to be deadpool and regenerate like fucking crazy you better be willing to shove a needle into your arm every four minutes while chugging down plates of pasta.

**Comment by Ratvar** [comment]
Alright I got it, a solution to genetics and virology. Remove.

**Comment by Pascal125** [comment]
First they came for Science, and i did not speak out.
Then they came for Medical, and i did not speak out.
Then they came for me, and all that was left were Assistants.

**Comment by sawrge** [comment]
First

**Comment by PKPenguin321** [comment]
https://www.youtube.com/watch?v=GJZRVG40Wwo

**Comment by Slignerd** [comment]
Honestly, I think a chance for people in crit to eventually recover if you don't finish them off can be equally frustrating and interesting, depending on the situation

**Comment by sawrge** [comment]
@Slignerd And thats why we have lings

**Comment by Iamgoofball** [comment]
@pascal125
Comparing removing shit game mechanics to Nazis murdering social groups makes you look like a whiny reddit poster

**Comment by SgtxRootbeer** [comment]
@sawrge yes, but lings revive from complete death, which is bullshit.
Crit though is different.

**Comment by KorPhaeron** [comment]
Will give XDTMs changes a chance to play out
